src/status_im2/navigation/screens.cljs

```diff
@@ -51,6 +52,11 @@
      :options   {:topBar {:visible false}}
      :component communities.discover/discover}
 
+    ;; THIS IS A TEMPORARY SCREEN. WILL REMOVE AFTER QA TESTING.
```

Comment:
I decided not to in order to keep this component and its API as isolated/pure as possible. The initial implementation of the component was kind of mixed with one of the community's screen. The result was that the API was not very reusable. The focus of this PR is on the smoothness of the animations regardless of what is rendered inside of it. And ofc following PRs that uses this component will be qa tested. (and also did not want to clutter the PR with changes not directly related to the component)

Reply:
>Ofc following PRs that uses this component will be qa tested. (and also did not want to clutter the PR with changes not directly related to the component)

It's fine, but you cluttered the PR with fake/temp stuff anyway, right? ;) The effort to create the fake screen could've been used to integrate with a real & simple screen. Maybe a `preview` screen would've been even better for QAs and reviewers to play around with this component, and to help out with future changes because you wouldn't need to throw away the code.

But anyways @OmarBasem, thanks for the explanation.

Reply:
Not the same effort really :) I tried with the messages screen, but considerable amount of changes would need to be done to the styling (lots of z-indexes which I do not know, and would need some debugging)

Reply:
Yeah, I wonder should we not have this as a component in quo2 library? 🤔 and then it has a preview screen too. I agree Omar it's easier to keep the API abstracted when it's not being used for it's implementation on creation 👍

Reply:
Yeah maybe I can move it to quo2